### ISSUE TITLE
Having a pip installable file(like requirements.txt) that can be used directly(without installing pixi or conda) inside docker environment.

### ISSUE THREAD

**Issue by raceychan** [prefix-dev/pixi#2172]
### Problem description

I use pixi for managing my project dependencies, However, it seems redundant to Install pixi inside a docker container.

1. docker container is an isolated environment, so installing dependencies is relatively 'safe'
2. by using pixi in local development, I already have a clean dependency lock file.
3. it feels difficult Installing pixi in docker using os like alpine 
4. pip freeze > requirements.txt  would only list dependencies in current conda environment.

Feature Request:
Having a pip installable file(like requirements.txt) that can be used a docker environment. 

I noticed that in recent version of pixi
'pixi project export' is introduced, however, AFAIK, it only exportx conda environment.yaml files, but I don't want to include conda inside my docker image, same reasons as 1,2,3 listed above.

**Comment by synapticarbors** [comment]
`pixi project export` also supports explicit conda specs that don't require a solve and can be installed via micromamba, which is very lightweight. I also started a draft of exporting a pip requirements file in https://github.com/prefix-dev/pixi/pull/2049, but that will only be for pypi dependencies, not conda packages.

**Comment by raceychan** [comment]
> `pixi project export` also supports explicit conda specs that don't require a solve and can be installed via micromamba, which is very lightweight. I also started a draft of exporting a pip requirements file in #2049, but that will only be for pypi dependencies, not conda packages.

Yeah, it is just that it takes some extra effort to install a package manager inside docker container, most importantly, package managers does not do much inside docker.

I was expecting that it might be a little tricky to export a requirements.txt when I am relying on conda dependencies.
 
I am comfortable switching to pypi-dependencies, but just ouf of curiousity:
Since we can already export conda environment to a requirements.txt by "pip freeze", does that mean it would automatically 'translate' dependencies we installed via "conda install" to pypi dependencies, or it just ignores them?

**Comment by olivier-lacroix** [comment]
No it won't @raceychan .

There is likely a more idiomatic way to achieve what you want (not including pixi in the docker image). I recommend you have a look at the [Docker example](https://github.com/prefix-dev/pixi/tree/main/examples/docker), which shows how to achieve that efficiently.

Let me know how you go 😃 !

**Comment by raceychan** [comment]
> No it won't @raceychan .
> 
> 
> 
> There is likely a more idiomatic way to achieve what you want (not including pixi in the docker image). I recommend you have a look at the [Docker example](https://github.com/prefix-dev/pixi/tree/main/examples/docker), which shows how to achieve that efficiently.
> 
> 
> 
> Let me know how you go 😃 !

The docker example you provided make more sense than the ones I found, would definitely try it out!
